config/locales/en.yml

```diff
@@ -1,6 +1,14 @@
 ---
 en:
+  password_validator:
```

Comment:
Hi @AyakorK, thank you for the PR! It works well, tne only thing is that those translation keys should also be added to the `fr.yml` file, as you can see in the screenshot below! 
<img width="609" alt="Capture d’écran 2022-11-07 à 14 58 07" src="https://user-images.githubusercontent.com/52420208/200329249-e6a4c693-a00d-4010-97a4-4ffcee0fe207.png">